pkg/storage/storage.go

```diff
@@ -195,7 +212,7 @@ func (is *ImageStore) GetRepositories() ([]string, error) {
 			return nil
 		}
 
-		is.log.Debug().Str("dir", path).Str("name", info.Name()).Msg("found image store")
+		//is.log.Debug().Str("dir", path).Str("name", info.Name()).Msg("found image store")
```

Comment:
Are you intending to uncomment this in the future?

Reply:
Yes, once we deploy and learn about how stable the new code is, may have to bring this back in.